README.md

```diff
@@ -1,16 +1,132 @@
-1. Currently, Collect is used to download blank forms, to fill them and to upload them to the server, there is no collaboration feature yet which help different enumerators to fill the same form with different sections.
-1. Sometimes organizations want to have some review of the forms before sending them to the server. Forms and instances from Collect instead of sending to the server will be sent to this app for the review. Forms from different enumerators of a team will be aggregated to a single device for review.
+### 1.2 Why Share<a name="why-share"></a>
+
+
+Share is designed to enable offline transfer of blank and filled forms. There are three major use cases for this multi-part surveys, supervisor review and blank form distribution.
+ODK Collect is designed in such a way that it allows enumerators to download the blank forms and provide them with the interface to fill the blank forms, but sometimes there can be more than one enumerators who want to fill the same form. Collect doesn’t allow more than one enumerators to collaborate on a single form. This is where Share comes into the picture using Share one can select any number of forms and can send to the other enumerator will all the resources attached to it.
+
+Also, sometimes organizations want to review filled forms before sending to the server. So enumerators can send the forms to the supervisor then the supervisor can mark forms as reviewed (accepted/rejected) with some feedback and then can send back to the same enumerator if required.
+
+In some areas, enumerators don’t have any internet connectivity, so they can’t fetch blank forms from the server. Share also allows to send a blank form in case of no internet connectivity provided the devices should be close enough to create a local connection.
+
+## 2. Installation and setup<a name="installation"></a>
+
+The Share app requires ODK Collect installed on the device first because it uses its database and storage for all its forms transfer.
+
+### 2.1 Install and setting up ODK Collect<a name="install-collect"></a>
+
+To install ODK Collect visit this link.
+To know more about ODK Collect visit the this documentation
+
+### 2.2 Install Share<a name="install-share"></a>
+
+Share app can be downloaded and installed from the link provided below.
+APK LINK [Add it here]
+
+## 3. Using Share<a name="using-share"></a>
+Share app can be used by enumerators in sharing forms and by supervisor in reviewing the forms.
+
+### 3.1 Viewing category of forms<a name="view-category"></a>
+
+Launch the app to see  a listing of all distinct versions of blank forms on the home screen. Tapping on any form will open a screen with four different tabs showing the statistics, sent, received and reviewed.
+
+To view details of any form category tap on any category shown on home screen.
+<p align="center">
+  <img src="https://github.com/lakshyagupta21/share/blob/91ad9210c67615263aaccced89616562f77249fa/screenshots/home_screen.png" width="280" height="500"/>
```

Comment:
These should be relative paths. We can change them later but wanted to make a note of it.